### ISSUE TITLE
Show Ammo Count in the Mechlab UI

### ISSUE THREAD

**Issue by skitthecrit** [MechWarriorOnline/issue-tracker#176]
<li><a href="https://github.com/MechWarriorOnline/issue-tracker/wiki/How-To-Upvote-An-Issue">How To Upvote This Issue</a></li>
<li><a href="https://github.com/MechWarriorOnline/issue-tracker/wiki/When-To-Add-A-Comment-To-An-Issue">When To Add A Comment To An Issue (vs Just Upvoting)</a></li>
<hr></hr>
Currently, the mechlab does not show how much ammo you have equipped for your weapons. This means it is difficult to know how much ammo you have equipped currently unless you manually count the tons of ammo currently on the 'mech, and account for any ammo quirks you may have. This is especially difficult for newer players, who may not know off the top of their head how much ammo per ton each weapon has.
<br></br>
Proposal: Add an extra value in the mechlab (most likely in the current mech information area) to let users know how much ammo they have equipped for their weapons, after all quirks are taken into account.

**Comment by Tyler799** [comment]
Definitely. I'd also like it to show an additional readout that accounts for the number of shells. Like 200 clan UAC/20 ammo would really be 50 "shots". (Because there are 4 shells per each shot)

I think that's doable with RACs as well, though it's not as useful. With multi-shell UACs it's pretty essential.

**Comment by skitthecrit** [comment]
That would be very useful information as well. Maybe have it so it has shots and total actual ammo displayed something like this 50(100), where the number outside of parentheses is total shots remaining, and the number inside is actual ammo count. One issue may be when you use half tons of ammo and you get a shot at the end that is not the whole burst, but they could just round it to the nearest whole number to compensate for that.

**Comment by dlikhten** [comment]
Piggy backing on @Tyler799 and @skitthecrit 

Well this combined with things like:

- cU-AC20 = 3-shell burst per shot
- cU-AC20 = should show ammo / 3. Why would anyone care to see what the total ammo count is, I can't elect to shoot 1/3 shells. Always show in shots.

LRM is different because LRM ammo is shared amongst all types of LRMs. But I'd still show it in "shots". (having an LRM 20 and LRM5 with 40 missiles should show 2 / 8 respectively, but shooting a 20 will change the numbers to 1 / 4 respectively)

This would make life so much easier in mechlab and in match.

**Comment by Tarogato-MWO** [comment]
> I can't elect to shoot 1/3 shells. Always show in shots.

Actually, if you overheat you can fire 1/3 of a shot. And if you double-tap too quickly with a UAC it won't fire all of the shells. But I agree that it would be nice to show the "trigger pulls" count instead of raw ammo count.

**Comment by dlikhten** [comment]
Those all sound like bugs actually... Maybe maaaaybe not the heat one.
